toolkits/global/packages/global-popup/js/popup.js

```diff
@@ -7,15 +7,15 @@ const Popup = class {
 		this._content = document.querySelector(`#${this._id}`);
 		this._className = 'c-popup';
 		this._isOpen = false;
-		this._expander = new Expander(this._trigger, this._content, {FOCUS_EVENT: true});
 		this._arrowClass = `${this._className}__arrow`;
 		this._closeClass = `${this._className}__close`;
-		this._closeButton = `<a href="javascript:;" class="${this._closeClass}">Close</a>`;
+		this._closeButton = `<button class="${this._closeClass}"><span class="u-visually-hidden">Close</span></button>`;
```

Comment:
I don't think utilities should technically be used in components (unless they're _defaults_ and configurable). It forces the app to import the utility classes. You can include the mixin in the popup css